Add icons for CCB App.

slushman opened the pull request:
BREAKING CHANGE: renamed ArrowDropDown and ArrowDropup to TriangleDown and TriangleUp for consistency. Two other icons are actual arrows, so it makes more sense to rename these to avoid future confusion.

slushman committed:
Add icons for CCB App.

alexbeckwith commented:
@slushman could we make the all caps ones pascal case and rename the Triangle ones to Caret instead? And rename Play/PlayOutline to Play/PlayFilled?

slushman committed:
Change names to all Pascal case. Rename Play icons. Remove Favorite since its a duplicate of Serve.

slushman commented:
> @slushman could we make the all caps ones pascal case and rename the Triangle ones to Caret instead? And rename Play/PlayOutline to Play/PlayFilled?

Renamed all icons to Pascal case. Renamed Play buttons. Remove Favorite since its the same as Serve. Left Triangle icons since they are triangle, not carets.
